packages/holocron/src/holocronModule.jsx

```diff
@@ -108,36 +153,52 @@ export default function holocronModule({
 
     HolocronModuleWrapper.propTypes = {
       load: PropTypes.func,
+      dispatch: PropTypes.func.isRequired,
     };
 
     HolocronModuleWrapper.defaultProps = {
       load: undefined,
     };
 
-    HolocronModuleWrapper.displayName = WrappedComponent && `HolocronModule(${
-      WrappedComponent.displayName || WrappedComponent.name || name
-    })`;
+    HolocronModuleWrapper.displayName = getDisplayName(getName(WrappedComponent, name));
 
-    HolocronModuleWrapper[REDUCER_KEY] = reducer;
     if (load && (options.ssr || global.BROWSER)) {
       HolocronModuleWrapper[LOAD_KEY] = load;
     }
-    hoistStatics(HolocronModuleWrapper, WrappedComponent);
 
-    const getModuleState = createSelector(
-      (state) => state.getIn(
-        [MODULES_STORE_KEY, name],
-        reducer(undefined, { type: INIT_MODULE_STATE })
-      ),
-      (moduleState) => moduleState.toJS()
-    );
+    let mapModuleStateToProps;
 
-    const mapModuleStateToProps = reducer && ((state) => {
-      const moduleState = getModuleState(state);
-      return { moduleState };
-    });
+    if (reducer && !name) {
+      console.warn(`Holocron Config API requires a 'name' when a 'reducer' is set for Module ${getDisplayName(getName(WrappedComponent, name))}.`);
```

Comment:
The warning should also indicate what the consequences of this are

Reply:
Fixed in 8aefa2b